L1Trigger/L1TTrackMatch/test/vtxPlotter.C

```diff
@@ -0,0 +1,766 @@
+#include "TROOT.h"
```

Comment:
Think my preference would have been to have a plotter written in python - but ok if it works for now